packages/pluggableWidgets/datagrid-web/src/Datagrid.editorPreview.tsx

```diff
@@ -39,9 +39,9 @@ export function preview(props: DatagridPreviewProps): ReactElement {
                         case "attribute":
                             return renderWrapper(
                                 <span className="td-text">
-                                    {"{"}
-                                    {column.attribute}
-                                    {"}"}
+                                    {"["}
+                                    {column.attribute.length > 0 ? column.attribute : "No attribute selected"}
```

Comment:
Might be lost in the extremely large snapshot diff, but no test that verifies this?

Reply:
The snapshots should not be affected by this file, as this is only for Design mode